src/components/ViewerTitle.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+import { ModelInfo } from '../constants';
+
+import '../../css/viewer_title.css';
+
+interface ViewerTitleProps extends ModelInfo {
+  trajectoryTitle?: string;
+}
+
+const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
+  props: ViewerTitleProps
+): JSX.Element => {
+  const { title, trajectoryTitle } = props;
+
+  const hasMetaData = (): boolean => {
+    for (const key in props) {
+      if (props[key as keyof ViewerTitleProps] !== undefined) {
+        return true;
```

Comment:
this needs to still ignore `trajectoryTitle` since it's not part of metadata

Reply:
ack, you're right, fixed